Tools/autotest/arducopter.py

```diff
@@ -9043,7 +9043,6 @@ def tests2b(self):  # this block currently around 9.5mins here
             self.RefindGPS,
             Test(self.GyroFFT, attempts=8),
             Test(self.GyroFFTHarmonic, attempts=8),
-            self.GyroFFTAverage,
```

Comment:
This is better inserted in the `disabled_tests` method which is also in `arducopter.py`.  With a link to an issue saying why it was disabled....